packages/graph-rag-example-helpers/src/graph_rag_example_helpers/datasets/animals.py

```diff
@@ -0,0 +1,27 @@
+import json
+
+import requests
+from langchain_core.documents import Document
+
+ANIMALS_JSONL_URL = "https://raw.githubusercontent.com/datastax/graph-rag/refs/heads/main/data/animals.jsonl"
+
+
+class Animals:
```

Comment:
Why a class and not just a function?

```
def load_animal_docs() -> list[Document]:
  ...
```

OR even `fetch` to imply the download and load?